Makefile

```diff
@@ -0,0 +1,335 @@
+	endif
+	ifeq ($(UNAME_S),msys*)
+		OSFLAG += Windows
+	endif
+	ifeq ($(UNAME_S),cygwin*)
+		OSFLAG += Windows
+	endif
+	ifeq ($(UNAME_S),MING*)
+		OSFLAG += Windows
+	endif
+
+## Print a variable EG: [make print-HOME]
+# print-<VAR>
+print-% : ; $(info $* is a $(flavor $*) variable set to [$($*) ]) @true
+
+############################### App parameters #################################
+APP := freqtrade
+# Looks for .env in 3 possible locations
+# Creates variables from .env
+ifneq (,$(wildcard $(HOME)/.$(APP)/.env))
+	include $(HOME)/.$(APP)/.env
+endif 
+ifneq (,$(wildcard user_data/.env))
+	include user_data/.env
+endif
+ifneq (,$(wildcard user_data/user_repo/.env))
+	include user_data/user_repo/.env
+endif
+# Define default path to App storage directory
+CONFIG_PATH := $(CONFIG_PATH)
+ifeq ($(CONFIG_PATH),)
+	CONFIG_PATH = $(HOME)/.$(APP)
+endif
+
+# Define location of git repo for user content
+USER_REPO := $(USER_REPO)
+ifeq ($(USER_REPO),)
+	USER_REPO = 
+endif
+
+# Define location of custom strategies
+STRATEGY_DIR := $(STRATEGY_DIR)
+ifeq ($(STRATEGY_DIR),)
+	STRATEGY_DIR = $(PROJECT_DIR)/user_data/strategies
+endif
+
+# Define Dockerfile
+APP_DOCKERFILE := $(APP_DOCKERFILE)
+ifeq ($(APP_DOCKERFILE),)
+	APP_DOCKERFILE = $(PROJECT_DIR)/Dockerfile
+endif
+
+# Define default strategy for docker commands
+DEFAULT_STRATEGY := $(DEFAULT_STRATEGY)
+ifeq ($(DEFAULT_STRATEGY),)
+	DEFAULT_STRATEGY = TestStrategy
+endif
+
+# Define additional args for docker start
+START_ARGS := $(START_ARGS)
+ifeq ($(START_ARGS),)
+	START_ARGS := -s $(DEFAULT_STRATEGY) --strategy-path $(STRATEGY_DIR) \
+		-c user_config/config.json
+endif
+
+####################### App Setup Commands #####################################
+OBJS := $(addprefix $(CONFIG_PATH)/,tradesv3.sqlite tradesv3.dryrun.sqlite \
+	localtime config.json ) $(USER_DIR)
+
+## Create files to start app if they don't exist
+setup-create: $(OBJS)
+$(CONFIG_PATH)/%.json: $(PROJECT_DIR)/config.json.example
+	cp -n config.json.example $(CONFIG_PATH)/config.json
+	cat /etc/localtime > $(CONFIG_PATH)/localtime
+
+$(CONFIG_PATH)/%.sqlite:
+	touch $(CONFIG_PATH)/tradesv3.sqlite
+	touch $(CONFIG_PATH)/tradesv3.dryrun.sqlite
+
+$(OBJS): | $(CONFIG_PATH)
+$(CONFIG_PATH):
+	mkdir $(CONFIG_PATH)
+
+user-repo:
+	@if [ ! -d user_data/user_repo/.git ] ;\
+	then \
+		echo cloning $(USER_REPO) ; \
+		git clone $(USER_REPO) user_data/user_repo; \
+		git checkout user_data/strategies/__init__.py; \
+		cp user_data/strategies/__init__.py $(STRATEGY_DIR)/; \
+		git checkout user_data/strategies/test_strategy.py; \
+		cp user_data/strategies/test_strategy.py $(STRATEGY_DIR)/; \
+	else \
+		echo "already exists"; \
+	fi
+$(USER_DIR): user-repo
+
+## Delete setup files ** ALL DATA WILL BE DELETED **
+setup-delete:
+	-rm $(CONFIG_PATH)/config.json 2> /dev/null
+	-rm $(CONFIG_PATH)/tradesv3.sqlite 2> /dev/null
+	-rm $(CONFIG_PATH)/tradesv3.dryrun.sqlite 2> /dev/null
+	-rm $(CONFIG_PATH)/localtime 2> /dev/null
```

Comment:
why is a object "localtime" even created? 

this seems odd to me and i've never seen it (unless i had some docker-command messed up).